### ISSUE TITLE
508-defect-3: Supplemental Claims | Add header to start page question

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#54260]
### Point of contact

Josh Kim

### Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

### Details

The `/start` page includes a `va-radio` asking "What type of claim are you filing a Supplemental Claim for?" which is inside a `legend`, but this question should be more discoverable by screen reader users if the text were wrapped inside an `H2`.

### Reproduction steps

1. Go to https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start
2. Using the screen reader headers menu will find the unique `h1`, and the `h2`s in the footer and site footer

### Proposed solution or next steps

Add `label-header-level="2"` to the `va-radio` web component to wrap the text inside an `H2`

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by Mottie** [comment]
Not quite ready for validation... I need to merge in another PR

**Comment by joshkimux** [comment]
<img width="1504" alt="Screenshot 2023-03-02 at 9 32 11 AM" src="https://user-images.githubusercontent.com/14154792/222457598-f13c3d6b-a9f3-4fa0-a5c3-140633c148c2.png">

Looking good!
